dts/bindings/bluetooth/st,hci-stm32wb0.yaml

```diff
@@ -0,0 +1,13 @@
+description: Bluetooth HCI driver for ST STM32WB0 series
+
+compatible: "st,hci-stm32wb0"
+
+include: bt-hci.yaml
+
+properties:
+  bt-hci-name:
+    default: "HCI over RAM"
+  bt-hci-bus:
+    default: "BT_HCI_BUS_VIRTUAL"
+  bt-hci-quirks:
+    default: ["BT_HCI_QUIRK_NO_RESET"]
```

Comment:
Note: you'll need to update these when #79878 has been merged (or the other way around, depending on which PR gets merged first)